build/PackageDiffIgnore.xml

```diff
@@ -1874,6 +1874,8 @@
 		</Fields>
 
 		<Properties>
+			<Member fullName="Windows.UI.Xaml.DependencyProperty Windows.UI.Xaml.FrameworkElement::StretchAffectsMeasureProperty()" reason="Not in WinUI. Made a regular property" />
+			<Member fullName="Microsoft.UI.Xaml.DependencyProperty Microsoft.UI.Xaml.FrameworkElement::StretchAffectsMeasureProperty()" reason="Not in WinUI. Made a regular property" />
```

Comment:
Let's keep this change for the next major

Reply:
This property is meant for internal use, and even if someone is using it, they'll be likely using the actual instance property, not this one.

I can still restore though